spring-cloud-gcp-docs/src/main/asciidoc/config.adoc

```diff
@@ -0,0 +1,115 @@
+== Spring Cloud Config
+
+Spring Cloud GCP makes it possible to use the
+https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/[Google
+Runtime Configuration API] as a
+https://cloud.spring.io/spring-cloud-config/[Spring Cloud Config] server to remotely store your
+application configuration data.
+
+The Spring Cloud GCP Config support is provided via a Spring Boot starter that makes notation like
```

Comment:
I'm not a fan of using the word "notation" here.
Here's how I would rephrase:

The Spring Cloud GCP Config support is provided via a Spring Boot starter that enables the use of the `@Value` annotation for injecting values stored in the Google Runtime Configuration API into fields of Spring-managed beans.